movo_common/movo_ros/src/movo_joint_interface/helpers.py

```diff
@@ -47,12 +48,19 @@ def dottedQuadToNum(ip):
 
 def get_ip_address(ifname):
     s = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
-    return socket.inet_ntoa(fcntl.ioctl(
+    struct_pack = struct.pack('256s', bytes(ifname[:15], encoding='utf-8'))
+    fctnl_ioctl = fcntl.ioctl(
         s.fileno(),
         0x8915,  # SIOCGIFADDR
-        struct.pack('256s', ifname[:15])
-    )[20:24])
-    
+        struct_pack
+    )
+    skt_inet_ntoa = socket.inet_ntoa(fctnl_ioctl[20:24])
+    if str(skt_inet_ntoa) == "129.97.71.49":
```

Comment:
I think there is a commit on Movo1 that Alex did that should be pushed and changes the compare statements to check for "10.66.171.1/2"